code/modules/mob/living/living.dm

```diff
@@ -402,9 +402,9 @@
 		for(var/obj/item/weapon/storage/S in src.contents)	//Check for storage items
 			L += get_contents(S)
 		for(var/obj/item/clothing/suit/storage/S in src.contents)//Check for labcoats and jackets
-			L += get_contents(S)
+			L += get_contents(S.hold)
 		for(var/obj/item/clothing/accessory/storage/S in src.contents)//Check for holsters
-			L += get_contents(S)
+			L += get_contents(S.hold)
```

Comment:
I was going to suggest that you define `get_contents()` for those two things as `return get_contents(hold)`, but then I remembered that `get_contents()` is already such a tremendous clusterfuck that it doesn't really matter

<sub>I'm pretty sure I agreed at some point to unfuck it but oops</sub>